wave_matching/tests/ndt_tests.cpp

```diff
@@ -0,0 +1,91 @@
+// Tests for wrapper
+
+#include <gtest/gtest.h>
+#include <pcl/io/pcd_io.h>
+#include <Eigen/Dense>
+#include <iostream>
+
+#include "wave/matching/ndt.hpp"
+
+namespace wave {
+
+// Fixture to load same pointcloud all the time
+class NDTTest : public testing::Test {
+ protected:
+    NDTTest() {}
+
+    virtual ~NDTTest() {
+        if (this->matcher) {
+            delete this->matcher;
+        }
+    }
+
+    virtual void SetUp() {
+        this->ref = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+        this->target = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+        pcl::io::loadPCDFile("data/testscan.pcd", *(this->ref));
+    }
+
+    void setParams(const float res, const Eigen::Affine3d perturb) {
```

Comment:
this does a lot more than just setParams.  Perhaps "initializePointcloud" or something is a better name?